process/smartContract/process.go

```diff
@@ -1457,6 +1457,14 @@ func (sc *scProcessor) createSmartContractResults(
 	txHash []byte,
 ) (bool, []data.TransactionHandler) {
 
+	if bytes.Equal(outAcc.Address, vm.StakingSCAddress) {
+		storageUpdates := process.GetSortedStorageUpdates(outAcc)
```

Comment:
why do we need to sort the output accounts only in the case of staking address? Should we sort them always?

Reply:
storage updates / not output accounts - we need this in order for all validators to create the same smart contract result. The created SCR is used in stakingToPeer